hudi-common/src/main/java/org/apache/hudi/metadata/HoodieMetadataPayload.java

```diff
@@ -209,9 +210,10 @@ public class HoodieMetadataPayload implements HoodieRecordPayload<HoodieMetadata
   private HoodieMetadataBloomFilter bloomFilterMetadata = null;
   private HoodieMetadataColumnStats columnStatMetadata = null;
   private HoodieRecordIndexInfo recordIndexMetadata;
+  private boolean isDeletedRecord = false;
 
   public HoodieMetadataPayload(GenericRecord record, Comparable<?> orderingVal) {
```

Comment:
to make it highlighted

```suggestion
  public HoodieMetadataPayload(@Nullable GenericRecord record, Comparable<?> orderingVal) {
```